src/button/component.jsx

```diff
@@ -187,6 +188,15 @@ export let Button : Component<ButtonOptions> = create({
             queryParam: true
         },
 
+        isFundingThrottleEnabled: {
+            type:       'boolean',
+            queryParam: true,
+            required:   false,
+            def() : boolean {
+                return fundingLogoThrottle && fundingLogoThrottle.isEnabled() ? true : false;
```

Comment:
So -- given that the order that `def` and `decorate` is called is a little non-deterministic, let's do this. In both places let's do:

```javascript
fundingLogoThrottle = fundingLogoThrottle || buildFundingLogoThrottle(normalizeProps(props));
```

-- then we can be sure that the throttle will exist when we need it in both places.

Reply:
Sure thing.